### ISSUE TITLE
Merge Editor: Togglign word wrap should apply to all editors

### ISSUE THREAD

**Issue by joaomoreno** [microsoft/vscode#155280]
It was weird when I toggled word wrap in the merge editor yet only the editor I was focused on had it toggled. I kind of expected all editors to have it toggled. My rationale is that toggling word wrap is usually a symptom of having long lines and the chances that all three editors have long lines is high if a single editor has long lines.

**Comment by gjsjohnmurray** [comment]
I agree. Until recently word wrap only toggled one side of the diff editor. See #127183
